app/controllers/application_controller.rb

```diff
@@ -2,4 +2,25 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+
+  private
+  def current_user
+     @current_user ||= begin
+      id = session[:current_user_id]
+      
+      if id
+        User.find_by(id: id)
```

Comment:
I talked with Chris about this and we think it's a better idea to raise an exception if there's a weird id in the session. So I'll change this too